app/helpers/bibs_helper.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module BibsHelper
+  include TeacherSetsHelper
+  include MlnResponse
+  include MlnHelper
+
+  def validate_input_params(req_body, validate = false)
+    bnumber = req_body['id']
+    title = req_body['title']
+    physical_description = var_field('300', req_body)
+
+    req_body_params = []
+
+    if bnumber.blank?
+      req_body_params << { value: bnumber, error_msg: BIB_NUMBER_EMPTY[:msg]}
+    end
+
+    if title.blank? && validate
+      req_body_params << { value: title, error_msg: TITLE_EMPTY[:msg]}
+    end
+
+    if physical_description.blank? && validate
+      req_body_params << { value: physical_description, error_msg: PHYSICAL_DESCRIPTION_EMPTY[:msg]}
+    end
+
+    validate_empty_values(req_body_params) if req_body_params.present?
+  end
+
+
+  def var_field(marc_tag, req_body)
+    @req_body = req_body
```

Comment:
This line is a bit confusing.

It maybe easier to have `var_field_data` as a TeacherSet method, which calls `var_field` with the req_body.

So you'd have TeacherSetHelper#var_field_data like

```
def var_field_data(marc_tag, merge)
  var_field(@req_body, marc_tag, merge)
end
```

And try to avoid setting `@req_body` in _any_ helper methods

Reply:
Then the implementation of `var_field` would basically be the current implementation of `var_field_data` — it's flipping how those two methods work: 

```
def var_field(req_body, marc_tag, merge = true)
    if merge == true
      req_body['varFields'].detect { |hash| hash['marcTag'] == marc_tag }['subfields'].map { |x| x['content'] }.join(', ')
    else
      req_body['varFields'].detect { |hash| hash['marcTag'] == marc_tag }['subfields'].detect { |hash| hash['tag'] == 'a' }['content']
    end
  rescue StandardError
    nil
  end
```

Then line 11 wouldn't change, it just wouldn't set the `@req_body`